webviz_subsurface/containers/_reservoir_simulation_timeseries.py

```diff
@@ -279,7 +281,7 @@ def _user_download_data(
                     requested_data = get_time_series_delta_ens(
                         ensemble_paths=self.ensemble_paths,
                         time_index=self.time_index,
-                        column_keys=self.column_keys,
+                        column_keys=[vector],
```

Comment:
This does not work in portable as the stored output of these functions are not available.
A possible workaround is to use `self.column_keys` as function argument, then filter the resulting dataframe.
```python
                    requested_data = get_time_series_delta_ens(
                        ensemble_paths=self.ensemble_paths,
                        time_index=self.time_index,
                        column_keys=self.column_keys,
                        base_ensembles=self.base_ensembles,
                        delta_ensembles=self.delta_ensembles,
                        ensemble_set_name=self.title,
                    )[[vector, "DATE", "ENSEMBLE", "REAL"]]
```